tests/unit/lms/resources/_js_config/__init___test.py

```diff
@@ -664,6 +646,43 @@ def routes(self, pyramid_config):
         pyramid_config.add_route("auth_route", "/auth")
 
 
+@pytest.mark.usefixtures("application_instance_service")
+class TestAddDeepLinkingAPI:
+    def test_it_adds_deep_linking_v11(self, js_config, context):
+        context.lti_params = {
+            "content_item_return_url": sentinel.content_item_return_url,
+        }
+
+        js_config.add_deep_linking_api()
+
+        config = js_config.asdict()
+        assert config["filePicker"]["deepLinkingAPI"] == {
+            "path": "/lti/1.1/deep_linking/form_fields",
+            "data": {
+                "content_item_return_url": sentinel.content_item_return_url,
+            },
+        }
+
+    def test_it_adds_deep_linking_v13(self, js_config, context, application_instance):
+        application_instance.lti_registration_id = 100
```

Comment:
This works, but how it works isn't very clear. On first inspection this didn't look like it was related to the test. 

We could use a returnless fixture to make this more obvious? Maybe a comment is all that's required?

```python
    @pytest.mark.usefixtures("with_lti_13")
    def test_it_adds_deep_linking_v13(self, js_config, context):
        ...
        
    @pytest.fixture
    def with_lti_13(self, application_instance):
        # Make the application instance `lti_version` return "1.3.0"
        application_instance.lti_registration_id = 100    
```